src/services/llm/claude.py

```
@@ -29,6 +29,31 @@
+
+def _init_system_cache() -> None:
+    global _DEFAULT_STRIPPED, _DEFAULT_BLOCKS
+    from src.services.llm.prompt_builder import build_claude_system_blocks, build_system_prompt
+
+    _DEFAULT_STRIPPED = build_system_prompt("default")
+    _DEFAULT_BLOCKS = build_claude_system_blocks("default")
+
+
+def _get_system(system_prompt: str) -> str | list[dict]:
+    """Return cache-control blocks for the default prompt; plain string otherwise."""
+    global _DEFAULT_STRIPPED, _DEFAULT_BLOCKS
```

Flake8 (7.3.0)

[error] `global _DEFAULT_STRIPPED` is unused: name is never assigned in scope

(F824)